examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>May be remove the dots from end of each sentence in the bullets above, only keep the last one.</p><p> </p><p>shall we add any explanation (1-2 sentences) how we decided those 22 features are the most important features?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>